README.md

```diff
@@ -42,18 +42,46 @@ Gatsby is used for basic routing and static site generation.
 
 The content for the documentation section is in the `docs/` subdirectory of the Storybook monorepo: https://github.com/storybookjs/storybook/tree/next/docs.
 
-To run this app while editing those files, checkout both this repository and the monorepo, and link them from this app:
+To run this app while editing those files, checkout both this repository and the monorepo, then:
+
+### Inside the `frontpage` repository:
+
+Run the following command to extract the monorepo docs (replace `$branch` with the name of the branch you need to work with).
 
 ```
-yarn link-monorepo-docs path/to/monorepo
+yarn extract-monorepo-docs $branch
```

Comment:
the `$branch` part is correct but the rest should stay as is. There two workflows: one for editing content from a symmlink, the other to pull content from the monorepo

Reply:
Thanks for that additional context. The PR looks good to me. Do you want to see any additional changes before I merge it?

Reply:
Ahh, I hadn't seen the `sync` script added to the monorepo. That's about 100x times better than my suggestion above. 🙌